feature-utils/poly-look/src/react-components/banners/notificationBanner.jsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+
+import "./notificationBanner.css";
+
+import standard from "../../static/images/icons/xmark-standard.svg";
+import success from "../../static/images/icons/xmark-success.svg";
+import error from "../../static/images/icons/xmark-error.svg";
+import warning from "../../static/images/icons/xmark-warning.svg";
+
+export const types = {
+  standard: { class: "standard", icon: standard },
+  success: { class: "success", icon: success },
+  error: { class: "error", icon: error },
+  warning: { class: "warning", icon: warning },
```

Comment:
Please, check for accessibility. Even I find it difficult to distinguish them (as you age, you see less contrast)

Reply:
I have already checked them, @victorjanin is there a way that this could be improved?

We are talking about the notification styles: https://www.figma.com/file/qIrr4QJrmYGqVQHQoCECux/polyPod-design-system?node-id=2893%3A2680

Reply:
I was rather talking about the icons themselves; I can't comment on them since they are binaries.

Reply:
I don't see a difference between error and warning, both look reddish to me; or success and the other. And there's very little difference for me among the two, one is reddish-black and the other simply black. Maybe they're placeholders, but still.